ccm_web/client/src/components/InlineTextEdit.tsx

```diff
@@ -85,20 +83,29 @@ function InlineTextEdit (props: InlineTextEditProps): JSX.Element {
     }
   }
 
+  const renderEditButton = (): JSX.Element => {
+    if (!props.isSaving) {
+      return (<EditIcon className={classes.editIcon} fontSize='inherit' onClick={toggleEdit}/>)
+    } else {
+      return (<EditIcon className={classes.editIconDisabled} fontSize='inherit'/>)
+    }
+  }
```

Comment:
Another way to write this would be:

```
  const renderEditButton = (): JSX.Element => {
    return (
      <EditIcon
        className={props.isSaving ? classes.editIconDisabled : classes.editIcon}
        fontSize='inherit'
        onClick={props.isSaving ? undefined : toggleEdit}
      />
    )
  }
```

Not necessarily better, just thought I'd throw it out there.